### DISCUSSION TITLE
cs/tcs/toc/topic1/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#38] [General]
# cs/tcs/toc/topic1/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/cs/tcs/toc/topic1/

**Comment by birchtree2** [comment]
最后DFA转正则表达式的形式化描述可能类似Floyd算法？不知道想的对不对
```
for (k = 1; k <= n; k++) {
  for (x = 1; x <= n; x++) {
    for (y = 1; y <= n; y++) {
      f[x][y] = min(f[x][y], f[x][k] + f[k][y]);
    }
  }
}
```
这里+对应字符串拼接, min对应$\cup$. 而 $f[i][i]$ 的初始值对应kleen star（如果有）

**Comment by Adam-Sm1th** [comment]
你这这写的，一会中文一会英文看的难受。要么全英文要么全中文。

**Comment by Adam-Sm1th** [comment]
内容写的还是不错的
